web/package/cockpit-d-installer.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Thu Feb 16 12:56:24 UTC 2023 - Ladislav Slezák <lslezak@suse.com>
+
+- Integrate cockpit terminal application
```

Comment:
Please, add a reference to the PR (gh#yast/d-installer#426). It makes it easier to share the changes later in the blog.